### PR TITLE
beta program page

### PR DESCRIPTION
**Pull request by rr404**

### CODE REVIEW

**Comment by rr404** [comment]
page to review : https://pr-503.d1to60jd2gb6y6.amplifyapp.com/u/beta_program/

File: crowdsec-docs/unversioned/beta_program.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<p align="center">
+<img src='/img/beta_program_header.png' alt="CrowdSec Beta Program" title="CrowdSec Beta Program" width="100%" />
+</p>
+
+## About the Beta Program
+
+Join our Beta program to get early access to new features and functionalities for the CrowdSec Security Engine and Console. Your participation and feedback are key to CrowdSec’s community-driven approach. 
+
+Help us refine and improve our offerings and provide solutions that serve you and the community better. 
+To join the CrowdSec Beta program, click the [Beta opt-in option directly in the Console](https://app.crowdsec.net/settings).
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/beta_program.mdx:15]: Does this always direct to user settings? what if user is within an org?

File: crowdsec-docs/unversioned/beta_program.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+**Important note**: During the beta phase, this feature is exclusively available for the Nginx Remediation Component. (*link coming soon, when beta is out !*)
+
+#### Quick install guide
+
+To begin exploring the capabilities of our AppSec enabled components, follow our straightforward installation guide:
+*Quick install guide coming soon, when beta is out !*
+
+#### Things you should try out
+
+Don’t know where to start? Here’s a list of a few cool things you can do to turn CrowdSec into a WAF.
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/beta_program.mdx:44]: Dont say `cool`: 

```
Here is a list of practical steps you can take to convert CrowdSec into a Web Application Firewall (WAF).
```

File: crowdsec-docs/unversioned/beta_program.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+To begin exploring the capabilities of our AppSec enabled components, follow our straightforward installation guide:
+*Quick install guide coming soon, when beta is out !*
+
+#### Things you should try out
+
+Don’t know where to start? Here’s a list of a few cool things you can do to turn CrowdSec into a WAF. 
+Give it a try and don’t forget to share your feedback!
+
+- Install wapp-rules/configs
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/beta_program.mdx:47]: appsec-rules

File: crowdsec-docs/unversioned/beta_program.mdx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+- Write your custom rules
+- Customize your wapp-config to choose active rules
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/beta_program.mdx:49]: appsec-config

File: crowdsec-docs/unversioned/beta_program.mdx (hunk 1 above)
Comment (rr404) [reply crowdsec-docs/unversioned/beta_program.mdx:15]: the org path has the org name in the path. 
This URL does take you to personal settings where you'll see the opt in section

File: crowdsec-docs/unversioned/beta_program.mdx (hunk 3 above)
Comment (rr404) [reply crowdsec-docs/unversioned/beta_program.mdx:47]: haha my typo hid it from my search and replace :D

**Review by buixor — APPROVED** [review]

**Pull request merged**